### DISCUSSION TITLE
MONARC Translation

### DISCUSSION THREAD

**Discussion by K8Sudo** [monarc-project/MonarcAppFO#378] [Q&A] [answered]
Hi,

can someone give me some information about the current process with the translations using translate.monarc.lu. Here at least in the German translation some improvements have been made. However, translations are not yet visible in the MONARC installations. Is there a point here where the de.po is updated in the [ng_client ](https://github.com/monarc-project/ng-client/blob/master/po/de.po )repository?

Best regards

Marc

**Comment by ruslanbaidan** [comment, marked as answer]
Hi,

When you make some changes on translate.monarc.lu and the changes confirmed by admin, they automatically come to ng_client repo as a separate commit in the specific branch, called "feature/internationalization".

Your changes were [committed](https://github.com/monarc-project/ng-client/compare/master...feature/internationalization) 3 times, the last one was on Sep 3.

The next step is to merge the branch in the master.
The po file changes can be used by tagging the master branch of "ng_client" and updating MonarcAppFO repository.

We are planning to make a release soon with some fixes and the translations can be added to it.
